src/site/pages/accessibility-guide.md

```diff
@@ -141,14 +141,15 @@ The [WCAG 2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/Overview.html#conte
 
 
 ## Testing
-To keep testing simple and easy to complete, we ask that contributors complete the following tests to try to catch most of the accessibility issues that may be present:
-1. Test keyboard accessibility, and check that these requirements are met:
+Many accessibility issues can be found by doing a few simple checks:
+1. Use an accessibility audit tool to check for violations. If you are using PatternFly in your project, we recommend using [aXe: The Accessibility Engine](https://www.deque.com/axe/) to check for accessibility violations. If you are contributing to PatternFly, refer to our [README.md](https://github.com/patternfly/patternfly-next/blob/master/README.md) on how to run this tool.
+2. Test keyboard accessibility, and check that these requirements are met:
     - All functionality is keyboard accessible
     - Order of elements in the HTML and in the layout follow a logical order
     - Elements with focus are clearly visible
-2. Use the [WAVE browser extension from WebAIM](https://wave.webaim.org/extension/) to disable styles, then test the information architecture and presence of adequate text labels.
-3. Test with any screen reader available in your operating system
-4. Check color contrast for:
+3. Use the [WAVE browser extension from WebAIM](https://wave.webaim.org/extension/) to disable styles, then test the information architecture and presence of adequate text labels.
```

Comment:
There are many ways to disable page styles using built in browser/os settings, that said, it's slightly different depending on which browser and OS a user is running. Do you think it would be better to recommend a method of disabling styles using built in browser/system features, or suggest installing yet another browser plugin?

For example, see the comment in this SO article that illustrates a few ways of doing the same thing. https://stackoverflow.com/a/15280065/4118955

Just thinking it may be a bit much to recommend users install two separate extensions, but maybe it's not that big a deal? Do we need to instruct users how to do this in the first place? Some users may already be using other tools to do this same purpose like standard "Web Developer Tools" in chrome or other similar extensions. The less we have to explain the easier it becomes to quickly scan this document so just trying to trim any language where possible.

Reply:
This is a good point. I'll take a stab at this. I'm definitely trying to keep guidelines trim and easy to follow. I do see a difference between the README vs the A11y Guide. The README includes what we expect when you are contributing. The A11y Guide is meant to be informational, regarding what our goals are for patternfly accessibility, and with information to help guide both contributors and consumers when working on accessibility.